src/templates/common/benefitsHubLinks/index.test.tsx

```diff
@@ -1,32 +0,0 @@
-import { render, screen } from '@testing-library/react'
```

Comment:
Removed all BenefitsHubLink things as it became a common component (RelatedLink(s))